bridges/otelzap/core.go

```diff
@@ -116,7 +116,9 @@ func (o *Core) Enabled(level zapcore.Level) bool {
 func (o *Core) With(fields []zapcore.Field) zapcore.Core {
 	cloned := o.clone()
 	if len(fields) > 0 {
-		cloned.attr = append(cloned.attr, convertField(&cloned.ctx, fields)...)
+		var attrbuf []log.KeyValue
+		cloned.ctx, attrbuf = convertField(cloned.ctx, fields)
+		cloned.attr = append(cloned.attr, attrbuf...)
```

Comment:
should it not be something like the following?


```suggestion
		ctx, attrbuf := convertField(fields)
		if ctx != nil { 
			cloned.ctx = ctx
		}
		cloned.attr = append(cloned.attr, attrbuf...)
```